components/list/list-item-mixin.js

```diff
@@ -295,20 +297,24 @@ export const ListItemMixin = superclass => class extends ListItemDragDropMixin(L
 		});
 	}
 
-	_onFocusIn() {
+	_onFocusIn(e) {
 		this._focusing = true;
+		e.stopPropagation();
```

Comment:
These `stopPropagation()` calls are necessary so that outer/parent list-items don't appear focused when a list-item within a nested list is focused.